### DISCUSSION TITLE
Usb boot?

### DISCUSSION THREAD

**Discussion by craigyjp** [probonopd/MiniDexed#408] [Q&A]
Has anyone successfully booted MiniDexed from USB? Reason I'm asking is I'm embedding a pi 400 into a master keyboard and I want it to boot from USB rather than an SD card.

**Comment by probonopd** [comment]
Hi @craigyjp. I am not sure whether Circle (the framework MiniDexed is using) supports booting from USB at all and even in case it does, it is not specially supported/tested in MiniDexed. So it's probably best to use a SD card for booting, or asking over at https://github.com/rsta2/circle/discussions whether it can be made work and what we'd have to do for it.

**Reply by craigyjp** [reply]
Thanks, I've just asked over on the circle discussion page. I cannot get an SD card extension cable for love nor money here in the Ukraine.
